prvni-krucky/podminky/excs/jednoduche-podminky.md

```diff
@@ -19,4 +19,16 @@ Vstup nepovolen
 
 pokud uživatel zadá špatné heslo.
 
-3. Upravte dále program tak, že pokud uživatel zadá správné heslo, program se ho ještě zeptá na věk a pustí jej dál pouze pokud je starší 18ti let. Pokud uživatel zadá heslo špatně, už se ho na věk neptejte a ukončete program voláním funkce `exit()`.
+3. K vypsání textu `Vstup nepovolen` nyní použijte funkci `exit()` namísto funkce `print()`. Funkci `exit()` můžete použít stejně jako funkci `print()`, tj. text k výpisu zadejte do závorek jako vstup. Funkce `exit()` ale spolu s výpisem zprávy program ukončí a zajistí, že uživatel bez správného hesla nebude moci v aplikaci provádět žádné další akce. 
```

Comment:
Toto je podle mě zbytečné zesložitění. Učil bych raději:
```
print('Vstup nepovolen')
exit()
```
Podle mě tam není nutné používat zanořené větve ani předtím.

Reply:
@ZelenyMartin A máš vzorové řešení bez zanořené podmínky? Protože když implementuješ toto `Upravte dále program tak, že pokud uživatel zadá správné heslo, program se ho ještě zeptá na věk`, tak v první podmínce nemůžeš mít výpis "Smíš vstoupit", protože jsi ještě nezkontroloval věk. Takže bys tam fakticky neměl mít nic, protože program nic dalšího dělat nemá. Což nedává moc smysl.
Možná by tedy ještě šla udělat varianta v první podmínce vypsat pouze `Vstup nepovolen`, pokud je heslo špatné, a ukončit program. A text `Smíš vstoupit` řešit až poté, co je v programu podmínka na věk.

Reply:
```
input("Zadej uživatelské jméno: ")
heslo = input("Zadej heslo: ")

if heslo != "simsalabim":
    print("Vstup nepovolen")
    exit()

if int(input("Zadej věk: ")) >= 18:
    print("Smíš vstoupit")
else:
    print("Vstup nepovolen")
```

Těch možností je spousta. Klidně to někdo může řešit zanořenou podmínkou a vyhne se duplikace výpisu `print("Vstup nepovolen")`